apps/sensenet/src/components/field-controls/webhook-payload.tsx

```diff
@@ -106,35 +120,15 @@ export const WebhookPayload: React.FC<ReactClientFieldSetting<LongTextFieldSetti
     default:
       return (
         <>
-          <FormLabel style={{ transform: 'translate(0, 1.5px) scale(0.75)', transformOrigin: 'top left' }}>
+          <Typography variant="caption" gutterBottom={true}>
             {localization.webhooksPayload.payload}
-          </FormLabel>
-          <RadioGroup
-            aria-label="payload"
-            name="payload"
-            value={String(useDefault)}
-            onChange={(event) => setUseDefault(event.target.value === 'true')}>
-            <FormControlLabel
-              disabled
-              value="true"
-              control={<Radio color="primary" />}
-              label={localization.webhooksPayload.useDefault}
-            />
-            <FormControlLabel
-              disabled
-              value="false"
-              control={<Radio color="primary" />}
-              label={localization.webhooksPayload.customize}
-            />
-          </RadioGroup>
-          <TextareaAutosize
-            className={classes.textArea}
-            disabled
-            aria-label="minimum height"
-            rowsMin={10}
-            cols={70}
-            value={value}
-          />
+          </Typography>
+          <Typography variant="body1" gutterBottom={true}>
+            {useDefault ? localization.webhooksPayload.defaultPayload : localization.webhooksPayload.customize}
+          </Typography>
+          <Paper elevation={0} className={widgetClasses.container}>
+            {<pre className={classes.defaultPayload}>{value}</pre> || ''}
+          </Paper>
```

Comment:
```tsx
{useDefault ? 
  <Typography variant="body1" gutterBottom={true}>
    {localization.webhooksPayload.defaultPayload} 
  </Typography>
:
  <Paper elevation={0} className={widgetClasses.container}>
    <pre className={classes.defaultPayload}>{value}</pre>
  </Paper>
}
```

- rename classes.defaultPayload to classes.payload (it is the customized one)
- remove localization.webhooksPayload.customize